src/css/fss-reset-global.css

```diff
@@ -0,0 +1,25 @@
+/*
```

Comment:
Yes, FSS was removed from Infusion. I'm assuming that this was brought over with the markup for the exploration tool that was used in the preview. The exploration tool is still using a much older instance of Infusion.